core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -723,6 +723,13 @@ public String toString() {
           .setDefaultValue(19998)
           .setDescription("The port that Alluxio master node runs on.")
           .build();
+  public static final PropertyKey MASTER_SAFEMODE_WAIT =
+      new Builder(Name.MASTER_SAFEMODE_WAIT)
```

Comment:
Perhaps rename to `MASTER_WORKER_CONNECT_WAIT_TIME`?

Reply:
Done.